docs/guides/setting-url-wordpress-source-data..md

```diff
@@ -0,0 +1,114 @@
+# Setting the URL of the WordPress source of data
+
+The most important setting in a Frontity project is the WordPress installation that can be used as the source of data
+
+The format of the WordPress URL will be different depending on the type of WordPress installation used with the Frontity project so the type of WordPress installation may require different ways of setting this url in the Frontity settings
+
+The main properties in the settings of this package that are implied in the URL of the WordPress source of data are:
+
+- `state.source.url`:  The URL of the WordPress
+- `state.source.api`:  The URL of the WordPress REST API
+- `state.wpSource.isWpCom`:  A flag to indicate a special use case of wordpress.com sites (Personal or Premium plans)
+
+`state.source.api` can be set manually but it can be computed from the value of `state.source.url` in most of the cases
```

Comment:
I would remove the `state.source.api` mentions. **It should never be set manually by the end-users from now on.**

I would add `state.wpSource.prefix` because that one can be changed by the end-users if they use something different than `/wp-json`.

I would also add info about the default values and requirements.

```suggestion
- `state.source.url`:  The URL of the WordPress. Required.
- `state.wpSource.isWpCom`:  A flag to indicate a special use case of wordpress.com sites (Personal or Premium plans). It is not required for Free wordpress.com sites. Defaults to `false`.
- `state.wpSource.prefix`: The prefix of the API. Defaults to `/wp-json`. It is not used if `isWpCom` is `true`.
```